ui/app/components/container-image-tag/index.ts

```diff
@@ -0,0 +1,64 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import ApiService from 'waypoint/services/api';
+import { StatusReport } from 'waypoint-pb';
+
+interface Args {
+  statusReport: StatusReport.AsObject;
+}
+
+export default class DockerImageBadge extends Component<Args> {
```

Comment:
Shall we make the class name match the filename?

```suggestion
export default class ContainerImageTag extends Component<Args> {
```

Or we could drop the class name altogether like:

```suggestion
export default class extends Component<Args> {
```